render-utils.js

```diff
@@ -0,0 +1,45 @@
+export function renderBeanie(beanie) {
+    const li = document.createElement('li');
+    li.classList.add('card');
+
+    const img = document.createElement('img');
+    img.src = beanie.image;
+    img.alt = beanie.title;
+
+    const content = document.createElement('div');
+    content.classList.add('content');
+
+    const h2 = document.createElement('h2');
+    h2.textContent = beanie.title;
+
+    const attributes = document.createElement('p');
+    attributes.classList.add('attributes');
+
+    const animal = document.createElement('span');
+    animal.textContent = beanie.animal;
+
+    const subtheme = document.createElement('span');
+    subtheme.textContent = beanie.subtheme;
+
+    const astroSign = document.createElement('span');
```

Comment:
i think the astro sign in your view is crammed against something else. to fix this you can do `textContent = ' ${astroSign '` with spaces inside the back ticks